src/js/apps/admin/programs/sidebar/program-sidebar_app.js

```diff
@@ -0,0 +1,39 @@
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+import { LayoutView } from 'js/views/admin/programs/sidebar/programs-sidebar_views';
+
+export default App.extend({
+  onBeforeStart({ program }) {
+    this.program = program;
+
+    this.showView(new LayoutView({
+      program: this.program,
+    }));
+  },
+  viewEvents: {
+    'save': 'onSave',
+    'close': 'stop',
+    'delete': 'onDelete',
+  },
+  onSave({ model }) {
+    this.program.save(model.attributes)
+      .then(() => {
+        Radio.request('sidebar', 'close');
+      }, ({ responseJSON }) => {
+        // TODO: display errors in form
+        // const errors = this.program.parseErrors(responseJSON);
+      });
+  },
+  onDelete() {
+    this.program.destroy();
+    this.stop();
+  },
+  onStop() {
+    this.program.trigger('editing', false);
```

Comment:
we won't need this trigger.  This is what we use to highlight the action being edited in the list, but you can'd edit a program while seeing the list so we won't be using it.